src/relocatepayeedialog.cpp

```diff
@@ -80,12 +84,16 @@ void relocatePayeeDialog::CreateControls()
     wxStaticLine* lineTop = new wxStaticLine(this,wxID_STATIC
         , wxDefaultPosition, wxDefaultSize, wxLI_HORIZONTAL);
 
-    cbSourcePayee_ = new wxComboBox(this, wxID_BOTTOM, ""
+    cbSourcePayee_ = new wxComboBox(this, wxID_BOTTOM
+        , sourcePayeeID_ == -1 ? "" : Model_Payee::get_payee_name(sourcePayeeID_)
         , wxDefaultPosition, btnSize
-        , Model_Payee::instance().all_payee_names());
-    cbSourcePayee_->AutoComplete(Model_Payee::instance().all_payee_names());
+        , Model_Payee::instance().all_payee_names(true));
+    cbSourcePayee_->AutoComplete(Model_Payee::instance().all_payee_names(true));
```

Comment:
for example, all_payee_names(true) - what does true mean? (I know that info available in the function header).
It bad style of progtamming to use Boolean parameters. 
May be better used_payee_names() ?

Reply:
The problem is not boolean parameter, but `all_*_*(*_only)` - all should not have any filtering.
This method is quite simple so it can be duplicated.